### DISCUSSION TITLE
[Question][Hw4] What is the default permission in `create` command?

### DISCUSSION THREAD

**Discussion by m-spitfire** [prosyslab-classroom/cs348-information-security#710] [Q&A]
Name: Murad Bashirov

Hello. When a process uses `create` command, what is the default permission set on the object? Only `Perm_own` or `Perm_own` + `Perm_read` ?

I think it should be `Perm_own` by the principle of least privilege, but just making sure.

Thansk.

**Comment by m-spitfire** [comment]
I think it's just `Perm_own` as I passed the test cases with it.

**Comment by sujin0529** [comment]
Yes, what you understand is correct!

Thanks.

**Comment by m-spitfire** [comment]
Thanks.
